### PR TITLE
FDP-5 Change the background color

### PR DESCRIPTION
**Pull request by AVandana-1806**
https://ca-mmisds.atlassian.net/browse/FDP-5?oldIssueView=true

Updated the background color for Black to green

### CODE REVIEW

**Comment by bencarle** [comment]
Please use the imperative mood for the PR title. e.g. "Change the background color" instead of "Changed the background color"

**Comment by bencarle** [comment]
All commit messages should begin with the Jira issue Id.

File: src/App.js [hunk 1]
```diff
@@ -74,6 +74,7 @@ class App extends Component {
                         deborah<br/>
                         dave<br/>
                         robert<br/>
+                        aaron<br/>
```
Comment (bencarle) [inline src/App.js]: This change is not relevant to this Jira issue. Please remove from this PR.

Comment (AVandana-1806) [reply src/App.js]: @bencarle Done

**Pull request merged**